vagrant/forum/forumdb.py

```diff
@@ -1,31 +1,15 @@
-#
-# Database access functions for the web forum.
-# 
+# "Database code" for the DB Forum.
 
-import time
+import datetime
 
-## Database connection
-DB = []
+POSTS = [("This is the first post.", datetime.datetime.now())]
```

Comment:
I'm curious about your thought process here in using tuples over dicts. Is this just to reduce complexity?